cmd/cloudFoundryDeploy_test.go

```diff
@@ -1351,6 +1354,15 @@ func TestMtaExtensionCredentials(t *testing.T) {
 		assert.NoError(t, err)
 	})
 
+	t.Run("invalid chars in credential key name", func(t *testing.T) {
+		err := handleMtaExtensionCredentials("mtaext1.mtaext",
+			map[string]interface{}{
+				"test.*Cred1": "myCredEnvVar1",
+			},
+		)
+		assert.EqualError(t, err, "credential key name 'test.*Cred1' contains unsupported character. Must contain only [A-Z][a-z][0-9][-_]")
```

Comment:
see failing go unit test:

--- FAIL: TestMtaExtensionCredentials (0.00s)
   --- FAIL: TestMtaExtensionCredentials/invalid_chars_in_credential_key_name (0.00s)
      cloudFoundryDeploy_test.go:1363: 
         Error Trace:	cloudFoundryDeploy_test.go:1363
         Error:      	Error message not equal:
            expected: "credential key name 'test.*Cred1' contains unsupported character. Must contain only [A-Z][a-z][0-9][-_]"
            actual  : "credential key name 'test.*Cred1' contains unsupported character. Must contain only ^[-_A-Za-z0-9]+$"
               Test:       	TestMtaExtensionCredentials/invalid_chars_in_credential_key_name

Reply:
Outdated, `go test -timeout 30s -run '^TestMtaExtensionCredentials$' github.com/SAP/jenkins-library/cmd -v` works in the meantime.